test/cartesian/getTicks.spec.ts

```diff
@@ -31,7 +32,7 @@ const EXAMPLE_INPUT = {
 
 jest.mock('../../src/util/DOMUtils', () => ({
   // We mock string size measurement, because getStringSize else returns 0 in these tests.
-  getStringSize: jest.fn((text: string) => ({ width: text.length, height: 20 })),
+  getStringSize: jest.fn((text: string) => ({ width: text.length, height: 12 })),
```

Comment:
Reduce height to 12 so that the `value`s in the `ticks` didn't need to be as large since their width is equal to the number of characters.